### DISCUSSION TITLE
Compiling executables into a subdirectory

### DISCUSSION THREAD

**Discussion by foobiane** [pure-data/pd-lib-builder#82] [Q&A] [answered]
Hi all! Forgive me if this question is redundant-- I'm relatively new to using pd-lib-builder.

I was wondering if pd-lib-builder had the ability to compile its executables into a user-specified directory via a makefile flag. I'm currently writing an external that has a `bin` subdirectory, and I'd like to put any executables or object files that are generated into that folder. I couldn't seem to find anything in the `Makefile.pdlibbuilder` documentation about it, so I figured I'd ask here.

**Comment by umlaeute** [comment, marked as answer]
this is a duplicate of #2, and the answer is unfortunately: "no" (there's also a "maybe in the future" attached to it, but given that #2 is 9 years old i would not bet my money on it)

**Reply by foobiane** [reply]
Got it. Apologies for the redundancy :)
